src/Raven.Client/Documents/Subscriptions/SubscriptionState.cs

```diff
@@ -15,6 +17,7 @@ public class SubscriptionState : IDatabaseTask, IDatabaseTaskStatus
     {
         public string Query { get; set; }
         public string ChangeVectorForNextBatchStartingPoint { get; set; }
+        public Dictionary<string, string> NextBatchStartingPointChangeVectors { get; set; }
```

Comment:
please open an issue to test upgrading 5.0 to 6.0 that have not committed `AcknowledgeSubscriptionBatchCommand` and friends

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-18224

Reply:
ChangeVectorForNextBatchStartingPointPer.. ?

Reply:
done